lib/rules/at-rule-empty-line-before/__tests__/index.js

```diff
@@ -42,33 +42,43 @@ const sharedAlwaysTests = {
 
   reject: [ {
     code: "a {} @media {}",
+    fixed: "a {}\n\n @media {}",
```

Comment:
What if we use similar code? 

```
div {\n  a {}  @media {}\n}
```

Which is looks like:

```
div {
  a {}  @media {}
}
```

We have have two spaces before `a` and `@media`. In this example if we add `\n\n` before two spaces we get correct indentation. These two examples basically the same. There is some amount of spaces before node we're fixing, but we can't know if they are good of bad.